### PR TITLE
Remove interactive legend hack

### PR DESCRIPTION
**Pull request by mpickering**

### CODE REVIEW

**Comment by mpickering** [comment]
Wait until the other PR is merged before reviewing.

**Comment by mpickering** [comment]
CI now passing on this branch as well. Please review.

**Comment by BinderDavid** [comment]
I'll have a look at this PR this evening.

**Comment by mpickering** [comment]
Thanks, can you not squash the commits please when you merge it. I rebased it now on master.

File: src/Eventlog/HtmlTemplate.hs [hunk 1]
```diff
@@ -86,9 +88,9 @@ htmlHeader dat desc as =
         H.style  $ preEscapedToHtml milligram
         H.style  $ preEscapedToHtml normalizecss
       else do
-        script ! src "https://cdn.jsdelivr.net/npm/vega@5.4.0" $ ""
-        script ! src "https://cdn.jsdelivr.net/npm/vega-lite@3.3.0" $ ""
-        script ! src "https://cdn.jsdelivr.net/npm/vega-embed@4.2.0" $ ""
+        script ! src "https://cdn.jsdelivr.net/npm/vega@5.10.0" $ ""
+        script ! src "https://cdn.jsdelivr.net/npm/vega-lite@4.7.0" $ ""
+        script ! src "https://cdn.jsdelivr.net/npm/vega-embed@6.3.0" $ ""
```
Comment (BinderDavid) [inline src/Eventlog/HtmlTemplate.hs]: This should be version 6.3.2
Version 6.3.0 of vega-embed does not exist and `curl https://cdn.jsdelivr.net/npm/vega-embed@6.3.0` results in an error.

Comment (mpickering) [reply src/Eventlog/HtmlTemplate.hs]: Thanks, fixed in a more robust way by making sure the versions for local and external assets are the same.

**Pull request merged**